packages/backend/src/database/migrations/20240424131220_populate_slugs.ts

```diff
@@ -0,0 +1,45 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    const sanitizeSlug = (slug: string) =>
+        `REGEXP_REPLACE(
+            LOWER(
+                ${slug}
+            ),
+            '[^a-z0-9]+', '-', 'g'
+        )`;
+    // This will populate slugs only for charts in spaces, not charts in dashboards
```

Comment:
Charts in dashboards will have not have slugs. I think that makes sense, as we can `promote` the dashboard, and all its charts.